firstproject/src/main/java/com/store/manager/Wagon.java

```diff
@@ -0,0 +1,20 @@
+package com.store.manager;
+
+public class Wagon {
```

Comment:
discover https://projectlombok.org and integrate it to the project as maven dependency to make Wagon class more elegant

Reply:
ok